### ISSUE TITLE
Renaming issues

### ISSUE THREAD

**Issue by PhiLhoSoft** [tststs/atom-ternjs#224]
I could successfully rename a function parameter and a local variable. Good.
I wanted to rename something like `ctrl.foo` to `ctrl.fooo` (only the `foo` part) but the package silently fails.
I can understand it is a hard, if not impossible to do correctly, problem, although I expected a deeper analysis from Tern! :smile: And if `ctrl.foo` is a function, it is able to find the definition in the file.
But, at least, perhaps you should show a message telling if the operation is not possible, or if you don't find any occurrence, etc.

Another woe: after renaming a variable, if I want to go back, I have to undo each occurrence of the variable. If the Atom editor allows grouping of actions under the same undo step, I suggest to do this.
In other words, undo of a rename operation should be atomic, should revert all occurrences at once.

Thanks.
